src/Dialect/ONNX/ONNXOps.cpp

```diff
@@ -3245,10 +3245,44 @@ LogicalResult ONNXGatherOp::inferShapes(
 //===----------------------------------------------------------------------===//
 // ConstantOfShape
 //===----------------------------------------------------------------------===//
+static LogicalResult verify(ONNXConstantOfShapeOp op){
+  ONNXConstantOfShapeOpAdaptor operandAdaptor = ONNXConstantOfShapeOpAdaptor(op);
+
+  auto input = operandAdaptor.input();
+
+  if(!hasShapeAndRank(input)){
```

Comment:
remember to run clang-format to reformat the code (otherwise one of the build checks will fail).

Reply:
See also https://llvm.org/docs/CodingStandards.html#spaces-before-parentheses

Reply:
and https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements